src/domains.js

```diff
@@ -16,16 +16,6 @@ import {
 import { URI } from "./config";
 
 export function getPayPalLoggerDomain(): string {
-  if (__ENV__ === ENV.LOCAL) {
```

Comment:
This is an indirect way to be test-aware. We shouldn't be changing how we go about things like this. Environmental differences happen, but this is just avoiding the real approach done in `getPayPalDomain` for a simpler test

Reply:
This might change behavior in stage though unfortunately. Do you know if stage will still be the proper url? Stage is considered local if memory serves, which is because locally they had still logged to msmaster endpoints.

Reply:
or local is considered stage. by default, in the local env we send all requests to the stage url unless you override it with a proxy

Reply:
ohhhh interesting, ok then I can add this back!! False assumption/understanding on my part!

Reply:
Is there a reason we handle it so differently when it is stage?